unload_databricks_data_to_s3.py

```diff
@@ -76,26 +76,52 @@ def export_meta_data(event_count: int, partition_count: int):
     meta_data: list = [{'event_count': event_count, 'partition_count': partition_count}]
     spark.createDataFrame(meta_data).write.mode("overwrite").json(args.s3_path + "/meta")
 
+def validate_args(args):
+    if args.table_versions_map is None:
+        raise Exception("Argument --table_versions_map is None")
+    if args.data_type is None:
+        raise Exception("Argument --data_type is None")
+    if args.sql is None:
+        raise Exception("Argument --sql is None")
+    if args.secret_scope is None:
+        raise Exception("Argument --secret_scope is None")
+    if args.secret_key_name_for_aws_access_key is None:
+        raise Exception("Argument --secret_key_name_for_aws_access_key is None")
+    if args.secret_key_name_for_aws_secret_key is None:
+        raise Exception("Argument --secret_key_name_for_aws_secret_key is None")
+    if args.secret_key_name_for_aws_session_token is None:
+        raise Exception("Argument --secret_key_name_for_aws_session_token is None")
+    if args.s3_region is None:
+        raise Exception("Argument --s3_region is None")
+    if args.s3_path is None:
+        raise Exception("Argument --s3_path is None")
 
 if __name__ == '__main__':
```

Comment:
is it possible to add a valid command example for reference?

Reply:
yep, can add that